lib/distrobaker.py

```diff
@@ -788,8 +825,10 @@ def get_buildsys(which):
             logger.debug('Successfully authenticated with the destination koji instance.')
         if which == 'source':
```

Comment:
By setting `bsys['session_start_time'] = datetime.datetime.now()` before this `if`, the two separate `session_start_time` assignments below could be eliminated. (Or am I missing something?)

Reply:
Of course, thanks.